docs/docs/components/lifecycle-operator/keptn-apps.md

```diff
@@ -14,8 +14,8 @@ with the ability to handle extra phases.
 It can execute the pre-/post-deployment evaluations of a workload
 and run pre-/post-deployment tasks.
 
-In its state, it tracks the currently active `Workload Instances`
-(`Pod`, `DaemonSet`, `StatefulSet`, and `ReplicaSet` resources),
+In its state, it tracks the currently active `Workload Versions`
+(a `DaemonSet`, `StatefulSet`, or `ReplicaSet` resource),
```

Comment:
```suggestion
In its state, it tracks the currently active `KeptnWorkloadVersions`
(`DaemonSet`, `StatefulSet`, or `ReplicaSet`),
```

Reply:
That sentence doesn't really make sense in this way i think. The parentheses don't make sense for KeptnWorkloadVersions...

Reply:
So how should we have it? 🤷‍♀️

Reply:
what about tracks the current workloads (bla bla or bla) and at what version they are

Reply:
yeah that makes sense i think.